### PR TITLE
Update docker-compose.yml

### PR DESCRIPTION
**Pull request by gbeaven90**
In response to the ticket I opened: #53 

This has fixed my issue:

Removed repeating spark-worker-1,spark-worker-2 from the .yml file and replaced with `spark-worker` for reusability in the docker-compose command.

Also updated `spark:3` to `spark:latest` like the documentation specifies.

Now running `docker-compose up --scale spark-worker=10` creates 10 worker instances as expected:
image

### CODE REVIEW

File: 3/debian-10/docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: '2'
 
 services:
   spark:
-    image: docker.io/bitnami/spark:3
```
Comment (alvneiayu) [inline 3/debian-10/docker-compose.yml:5]: could you keep without change the docker.io/bitnami/spark:3, please?

File: 3/debian-10/docker-compose.yml [hunk 2]
```diff
@@ -11,8 +11,8 @@ services:
       - SPARK_SSL_ENABLED=no
     ports:
       - '8080:8080'
-  spark-worker-1:
-    image: docker.io/bitnami/spark:3
```
Comment (alvneiayu) [inline 3/debian-10/docker-compose.yml:15]: same here

File: 3/debian-10/docker-compose.yml [hunk 3]
```diff
@@ -21,15 +21,4 @@ services:
       - SPARK_RPC_AUTHENTICATION_ENABLED=no
       - SPARK_RPC_ENCRYPTION_ENABLED=no
       - SPARK_LOCAL_STORAGE_ENCRYPTION_ENABLED=no
-      - SPARK_SSL_ENABLED=no
-  spark-worker-2:
-    image: docker.io/bitnami/spark:3
```
Comment (alvneiayu) [inline 3/debian-10/docker-compose.yml:26]: same here

File: docker-compose.yml [hunk 4]
```diff
@@ -2,7 +2,7 @@ version: '2'
 
 services:
   spark:
-    image: docker.io/bitnami/spark:3
```
Comment (alvneiayu) [inline docker-compose.yml:5]: same here

File: docker-compose.yml [hunk 5]
```diff
@@ -11,8 +11,8 @@ services:
       - SPARK_SSL_ENABLED=no
     ports:
       - '8080:8080'
-  spark-worker-1:
-    image: docker.io/bitnami/spark:3
```
Comment (alvneiayu) [inline docker-compose.yml:15]: same here

File: docker-compose.yml [hunk 6]
```diff
@@ -21,15 +21,4 @@ services:
       - SPARK_RPC_AUTHENTICATION_ENABLED=no
       - SPARK_RPC_ENCRYPTION_ENABLED=no
       - SPARK_LOCAL_STORAGE_ENCRYPTION_ENABLED=no
-      - SPARK_SSL_ENABLED=no
-  spark-worker-2:
-    image: docker.io/bitnami/spark:3
```
Comment (alvneiayu) [inline docker-compose.yml:26]: same here

**Review by alvneiayu — CHANGES_REQUESTED** [review]
hi @gbeaven90 

First of all, thanks a lot for your PR and your time.

I included some review comments. I will test the change in our pipeline, after you apply the comments suggested.

thanks a lot

Álvaro

File: 3/debian-10/docker-compose.yml (hunk 3 above)
Comment (gbeaven90) [reply 3/debian-10/docker-compose.yml:26]: spark-worker-2 no longer exists, it's just spark-worker now, so this one doesn't apply.

File: docker-compose.yml (hunk 6 above)
Comment (gbeaven90) [reply docker-compose.yml:26]: spark-worker-2 no longer exists, it's just spark-worker now, so this one doesn't apply.

**Review by gbeaven90 — COMMENTED** [review]
Changed tag version from 'latest' back to '3' in all applicable file updates.

**Comment by alvneiayu** [comment]
hi @gbeaven90 

I didn't answer you because I am changing some test in our pipeline to land your change. I will come back as soon as possible.

Thanks a lot

Álvaro

**Review by alvneiayu — APPROVED** [review]
LGTM

**Pull request merged**